packages/shared/src/components/onboarding/CreateFeedButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { HTMLAttributes, ReactElement } from 'react';
+import classNames from 'classnames';
+import useFeedSettings from '../../hooks/useFeedSettings';
+import { Button, ButtonElementType } from '../buttons/Button';
+import { SimpleTooltip } from '../tooltips';
+import useMedia from '../../hooks/useMedia';
+import { tablet } from '../../styles/media';
+
+export type CreateFeedButtonProps = {
+  className?: string;
+  requiredTagsThreshold: number;
+} & Pick<HTMLAttributes<ButtonElementType<'button'>>, 'onClick'>;
+
+export const CreateFeedButton = ({
+  className,
+  requiredTagsThreshold,
+  onClick,
+}: CreateFeedButtonProps): ReactElement => {
+  const { feedSettings } = useFeedSettings();
+  const tagsCount = feedSettings?.includeTags?.length || 0;
+  const canCreateFeed = tagsCount >= requiredTagsThreshold;
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
```

Comment:
If it's just targeting mobile I prefer to use `sidebarRendered` from `useSidebar` mainly suggesting it as I want to create a new hook that will make the refactor there easier. (also more inline with how we evaluate atm for mobile, only tablet has the exception now)